packages/components/container/src/container.vue

```diff
@@ -18,6 +19,7 @@ const props = defineProps({
    */
   direction: {
     type: String,
+    validator: isDirectionType,
```

Comment:
```suggestion
    validator: (val: string) => ['horizontal', 'vertical'].includes(val),
```

Because the `direction` attribute is `'ltr' | 'rtl' | 'ttb' | 'btt'` in the ElDrawer component, moving the `validator` to the global may not be a good idea.

Reply:
@chenyuqiua Please adjust your code a bit as @tolking suggested.

Reply:
I have modified it as above @btea